### ISSUE TITLE
"Haunting"

### ISSUE THREAD

**Issue by ottomated** [ottomated/CrewLink#25]
Enhance the ghost's VC powers:
- Talking to living impostors, with audio distortion?
- Emit a quiet "haunting" sound effect to nearby living players

**Comment by Lavodan** [comment]
I'm not sure if this is a good idea, as it would mess with the balance of the game quite a bit, even if limited, you could easily instakill the imposters, unless the effects were cranked up to 11, and then whats the point of it anyways?
This would work in very controlled groups with a lot of self restraint, but I just dont see it; perhaps it coould be only as an optional... option by the host of the game.

**Comment by Peteledete** [comment]
> I'm not sure if this is a good idea, as it would mess with the balance of the game quite a bit, even if limited, you could easily instakill the imposters, unless the effects were cranked up to 11, and then whats the point of it anyways?
> This would work in very controlled groups with a lot of self restraint, but I just dont see it; perhaps it coould be only as an optional... option by the host of the game.

CrewLink as a whole is an unfair advantage lol

**Comment by Lavodan** [comment]
> CrewLink as a whole is an unfair advantage lol
I disagree, since only people within your proximity can hear you, its not like you can expose the imposters by screaming. The imposters also get an advantage by being able to lie/manipulate people more, and if they get a secure place they can even discuss their plans; not to mention planning double kills.
Having ghosts be able to talk just makes it literally impossible for imposters to do anything
Only additional thing that would nagate this for me would be if you could only hear the ghosts from the other team (imposter ghost with crew and crew ghosts with imposters) However this still raises some concerns and I dunno if its worth it.
I think it would be all round better to just stick to ghosts with ghosts and people with people

**Comment by francois07** [comment]
> I'm not sure if this is a good idea, as it would mess with the balance of the game quite a bit, even if limited, you could easily instakill the imposters, unless the effects were cranked up to 11, and then whats the point of it anyways?
> This would work in very controlled groups with a lot of self restraint, but I just dont see it; perhaps it coould be only as an optional... option by the host of the game.

But the idea is only for impostors to be able to hear ghosts though? How would that make it unbalanced?
